tools/livecd-creator

```diff
@@ -197,10 +207,19 @@ def main():
         logging.error("kickstart error: %s", e)
         return 1
 
-    if not ks.handler.repo.seen:
+    if not ks.handler.repo.seen and not options.repo:
         print("Kickstart (%s) must have at least one repository." % (options.kscfg), file=sys.stderr)
         return 1
 
+    if ks.handler.repo.seen and options.repo:
+        print("Ignoring kickstart (%s) repo definitions in favor of \"%s\"" %
+              (options.kscfg, options.repo), file=sys.stderr)
+
+    if options.gpgcheck and not options.repo:
+        print("Option --gpgcheck requires the use of --repo=PATH for dnf configuration",
```

Comment:
`dnf configuration` -> `repo configuration`

Reply:
Fixed in patch v3, fccafd3052.

Good catch on the commit message. That has also been fixed.